server/interceptor/interceptor_test.go

```diff
@@ -397,12 +397,13 @@ func TestTrillianInterceptor_QuotaInterception_ReturnsTokens(t *testing.T) {
 		},
 	}
 
-	// Use a ctx with deadline so we can check whether PutTokens inherits it.
-	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Minute)
+	// Use a ctx with a timeout smaller than PutTokensTimeout
+	ctx, cancel := context.WithTimeout(context.Background(), PutTokensTimeout-3*time.Second)
```

Comment:
Depending on the value of `PutTokensTimeout`, this might be a negative number. 
Consider using `PutTokensTimeout/2`

Reply:
Good call. I've set an explicit value to PutTokensTimeout for the duration of this test, so it's easy to reason about the numbers.